cram_moveit/src/collision-environment.lisp

```diff
@@ -266,6 +266,25 @@ bridge.")
            (moveit)
            "Removed `~a' from environment server." name))))))
 
+(defun clear-all-moveit-collision-objects ()
```

Comment:
Is this the deletion we talked about? Not specifying any objects, hence MoveIt! deletes all non-attached ones? Great!